File: app/src/main/java/com/narcyber/mvpbasics/application/MvpBasicApplication.java
package com.narcyber.mvpbasics.application;

import android.app.Application;
import android.util.Log;

import com.narcyber.mvpbasics.db.RoomDb;

public class MvpBasicApplication extends Application {


    @Override
    public void onCreate() {
        super.onCreate();
        RoomDb.getINSTANCE(this);
        Log.d("NAR", "main is created ");

    }
}

File: app/src/main/java/com/narcyber/mvpbasics/application/MvpBasicApplication.kt
package com.narcyber.mvpbasics.application

import android.app.Application
import com.narcyber.mvpbasics.db.RoomDb

class MvpBasicApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        RoomDb.getINSTANCE(this)
    }
}